### DISCUSSION TITLE
What is the reason for hard-coding `decode_times=False` in `ux.open_dataset()` and `ux.open_mfdataset()`?

### DISCUSSION THREAD

**Discussion by tomvothecoder** [UXARRAY/uxarray#764] [Q&A] [answered]
Hi, I'm just wondering why `decode_times=False` is hard-coded in `ux.open_dataset()` and `ux.open_mfdataset()`? 

https://github.com/UXARRAY/uxarray/blob/54da444302425be04d33abe914062d8ffc5fed35/uxarray/core/api.py#L170-L171
https://github.com/UXARRAY/uxarray/blob/54da444302425be04d33abe914062d8ffc5fed35/uxarray/core/api.py#L259-L260

I'm trying to use xCDAT's temporal [APIs](https://xcdat.readthedocs.io/en/latest/generated/xcdat.temporal.TemporalAccessor.html) on `uxarray.UxDataset` objects, but those APIs require time coordinates to be decoded as `datetime` or `cftime` objects (`decode_times=True`).

### Workaround
I was able to decode time coordinates using [`xarray.decode_cf()`](https://docs.xarray.dev/en/stable/generated/xarray.decode_cf.html) with `decode_times=True` and `use_cftime=True`. All downstream xCDAT temporal APIs worked.  

**However, the issue is that `xarray.decode_cf()` returns an `xarray.Dataset` object, rather than the `uxarray.UxDataset` object. This means all of the `uxarray` attributes and operators are no longer accessible on the returned object.**

**Comment by philipc2** [comment, marked as answer]
Hi @tomvothecoder 

This shouldn't be hard coded, thanks for the catch! Will handle this with #766

**Reply by tomvothecoder** [reply]
Thanks @philipc2!
